internal/provisioner/utility_group.go

```diff
@@ -90,9 +110,16 @@ func newUtilityGroupHandle(kops *kops.Cmd, provisioner *KopsProvisioner, cluster
 
 // CreateUtilityGroup  creates  and  starts  all of  the  third  party
 // services needed to run a cluster.
-func (group utilityGroup) CreateUtilityGroup() error {
+func (group utilityGroup) CreateUtilityGroup(kops *kops.Cmd) error {
```

Comment:
The utilityGroup has a member `kops` of type `*kops.Cmd`. Do we need to have `kops` passed in as an argument here (and in Update as well)? `kops` is instantiated in `newUtilityGroupHandle`, unless for some reason here we need a different instance of that type?

Reply:
Thanks for the catch, totally missed that. Fixing now :)